modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1224,16 +1222,12 @@
 			name = "license to [choice]"
 
 // Donation reward for 1ceres
-/obj/item/sequence_scanner/korpstech
-	name = "Korpstech genetics scanner"
-	desc = "A hand-held sequence scanner for analyzing someone's gene sequence on the fly. This one is bright pink and has some kind of Helix shape on its back."
-	icon_state = "korpsgenetic"
-	inhand_icon_state = "korpsgenetic"
-	worn_icon_state = "korpsgenetic"
-	icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
-	lefthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_left.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_right.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
+/obj/item/clothing/suit/jacket/gorlexharness
+	name = "engine technician harness"
+	desc = "A blood-red engineering technician harness. You can't seem to figure out an use to it, but it seems it's magnetized at places."
```

Comment:
```suggestion
	desc = "A blood-red engineering technician harness. You can't seem to figure out a use to it, but it seems to seal magnetically in some places."
```
`A use`, not `An use` ;)
Second part I might have misunderstood, I assume it's meaning magnetism as a function and not a property - if the latter, `but the plating seems magnetic in some places` would do as well

Reply:
Other than this small grammar note, it all checks out, sprites are badass too 👍

Reply:
Oh thanks, I almost missed out on those two little mistakes, ahaha. And, thanks!